docs/dev_notes.md

```diff
@@ -13,26 +13,28 @@ The build system has been recently revamped. Please see details in the file
 
 ### Unit Testing (rspec, puppet-lint, etc)
 
-| OS           | Ruby  | Puppet |
-|--------------|-------|--------|
-| RHEL 6       | 1.8.7 | 3.8.7  |
-| RHEL 7       | 2.0.0 | 3.8.7  |
-| Ubuntu 14.04 | 1.9.3 | 3.8.7  |
-| Ubuntu 16.06 | 2.3.1 | 3.8.5  |
+| OS            | Ruby  | Puppet |
```

Comment:
This is due to the fact that current unit tests in CI run only on Ubuntu 20.04 (latest) and not on RHEL7. It is specified here: https://github.com/StackStorm/puppet-st2/blob/master/.github/workflows/build.yaml#L15-L38

While we do run integrations tests for RHEL7, running units tests won't make much of a difference since it just serves as the base platform to setup ruby env and run the tests on.